src/components/Form.jsx

```diff
@@ -0,0 +1,16 @@
+import Button from "./Button";
+function Form(props){
+    return (
+        <form onSubmit={(e) => {
+            e.preventDefault();
+            const titleValue = e.target.querySelector('input[placeholder="Title"]').value;
+            const contentValue = e.target.querySelector('input[placeholder="Content"]').value;
+            props.onSubmit(titleValue, contentValue)
```

Comment:
This isn't the right way to do it, you can either access the name property on for the form through the event, 

or you can control the state of the inputs and use them instead of the query selector